libprobe/probe.py

```diff
@@ -211,7 +211,7 @@ def start(self):
         signal.signal(signal.SIGINT, self._stop)
         signal.signal(signal.SIGTERM, self._stop)
 
-        self.loop = asyncio.get_event_loop()
+        self.loop = asyncio.new_event_loop()
```

Comment:
I prefer to define the `loop` before "start". In case you want to use the `loop` before calling `start`.